cdk/lib/support-reminders.ts

```diff
@@ -70,12 +74,6 @@ export class SupportReminders extends GuStack {
 
 
 		// ---- API-triggered lambda functions ---- //
-		const searchRemindersLambda = new GuLambdaFunction(this, "search-reminders", {
-			handler: "search-reminders/lambda/lambda.handler",
-			functionName: `support-reminders-search-reminders-${this.stage}-CDK`,
-			...sharedLambdaProps,
-		});
-
 		const createRemindersSignupLambda = new GuLambdaFunction(this, "create-reminders-signup", {
 			handler: "create-reminder-signup/lambda/lambda.handler",
 			functionName: `support-reminders-create-reminder-signup-${this.stage}-CDK`,
```

Comment:
I think we also need to change the function names to match what's in the riff-raff.yaml